dashboards-observability/public/components/app.tsx

```diff
@@ -60,7 +60,7 @@ export const App = ({
                       chrome={chrome}
                       http={http}
                       notifications={notifications}
-                      parentBreadcrumb={parentBreadcrumb}
+                      parentBreadcrumbs={[parentBreadcrumb]}
```

Comment:
not related to the PR but i never understood why `parentBreadcrumb` needs to be passed through props previously, it's just a constant https://github.com/opensearch-project/observability/blob/6d3c351b6561da50961952e5a78fa4b2ec42202a/dashboards-observability/public/components/app.tsx#L38-L41